cmd/thor/main.go

```diff
@@ -150,7 +150,8 @@ func defaultAction(ctx *cli.Context) error {
 
 	defer func() { log.Info("exited") }()
 
-	initLogger(ctx)
+	levelVar := initLogger(ctx)
```

Comment:
@MakisChristou IMO we should just `initLogger(ctx)` and delete the `handleSIGHUP` as we haven't determined how we will change the logger yet